Add passing spec and revision to user show for story 19.

linda-le1 opened the pull request:
Co-Authored-By: Daniel "Danny" Moran  <danmoran-pro@users.noreply.github.com>

**Functionality:**
Add info to user profile show page.

**Testing**(RSpec tests and development environment): Passes 100% - looks good in local.

**Related Issues:**
Closes #12

linda-le1 committed:
Add passing spec and revision to user show for story 19.

Co-Authored-By: Daniel "Danny" Moran  <danmoran-pro@users.noreply.github.com>

madelynrr commented on spec/features/users/user_spec.rb:
```diff
@@ -16,4 +16,30 @@
+
+    user = create(:random_user, role: 0)
+
+    visit '/'
```
I really want to figure out why allow_any_instance_of isn't working for us.

madelynrr commented on spec/features/users/user_spec.rb:
```diff
@@ -16,4 +16,30 @@
+    expect(page).to have_content("#{user.email}")
+
+    expect(page).to have_link("Edit Profile")
+    save_and_open_page
```
Take out save and open page first!

linda-le1 committed:
Remove save and open page.

Co-authored-by: Daniel Moran 
<49169813+danmoran-pro@users.noreply.github.com>
Co-authored-by: Madelyn Romero 
<51763489+madelynrr@users.noreply.github.com>
Co-authored-by: Travis Borgsmiller 
<37959929+TravisBorgsmiller@users.noreply.github.com>
Co-authored-by: Jordan Holtkamp 
<53549145+jordanholtkamp@users.noreply.github.com>

Co-Authored-By: Daniel "Danny" Moran  <danmoran-pro@users.noreply.github.com>
